docker-compose.yml

```diff
@@ -2,7 +2,10 @@ version: '3'
 
 services:
   inbound:
-    image: local/mhs-inbound:${INBOUND_BUILD_TAG}
+    image: local/mhs-inbound:${BUILD_TAG}
+    # build:
```

Comment:
Is there a ticket in the backlog for what is still left to do?

Reply:
Yes - https://gpitbjss.atlassian.net/browse/NIAD-154